lib/debootstrap.sh

```diff
@@ -774,7 +774,9 @@ create_image()
 
 
 	if [[ -z $SEND_TO_SERVER ]]; then
-
+		# custom post_build_image_modify hook to run before fingerprinting and compression
+		[[ $(type -t post_build_image_modify) == function ]] && display_alert "Custom Hook Detected" "post_build_image_modify" "info" && post_build_image_modify "${DESTIMG}/${version}.img"
```

Comment:
🖖 all your hooks are belong to fragments

Reply:
yes... now I understand and  I've seen the light.

Reply:
yours is still a little over my head tho  🤓